lib/cmd/fh3/appdata/export/start.js

```diff
@@ -44,7 +44,15 @@ module.exports = {
         return cb(err);
       }
 
-      return cb(null, 'Created a new export job with ID: ' + result);
+      var info = [
+        'Created a new export job with ID: ' + result,
+        '. You can use',
+        ' `fhc appdata export status --envId=' + params.envId,
+        ' --appId=' + params.appId + ' --jobId=' + result,
+        '` to poll the job status'
```

Comment:
I suggest removing the backticks and trying to get the command on a separate line so it's easier for the user to copy:

```
Created a new ... To check on the job status you can use the following command:
fhc appdata export ...
```